### ISSUE TITLE
Notifications not appearing on Login or file upload

### ISSUE THREAD

**Issue by simality** [freelawproject/recap-chrome#19]
Popup notifications do not appear when on login, file upload or docket update.  
However:
The icon does turn blue
Files are uploading
Dockets do update (most of the time).

Additionally there is no notice when I go to purchase a document that has already been recapped.

I did some recapping with the dev tools enabled this evening.

Below is what it looks like after I "recap" a document.
![recap calls](https://cloud.githubusercontent.com/assets/5634271/7130521/c4496aac-e23f-11e4-990c-948d4d2e6735.PNG)

For the sake of thoroughness I found part of a docket (https://ecf.ilnd.uscourts.gov/cgi-bin/iqquerymenu.pl?284511) that hadn't been recapped and uploaded that.  

![plvi docket](https://cloud.githubusercontent.com/assets/5634271/7130633/de451396-e241-11e4-8131-caa78effc3b8.PNG)

This is what it looks like when a document menu (multiple documents in one upload) is recapped
![recap calls plvi doc 5](https://cloud.githubusercontent.com/assets/5634271/7130638/ef615e00-e241-11e4-9e15-f33ea8f6b15f.PNG)

And this is what it looks like when an unrecapped exhibit gets recapped
![plvi docket 5a](https://cloud.githubusercontent.com/assets/5634271/7130643/fadd1256-e241-11e4-9fe5-dce63d38c481.PNG)

And this is what it looks like when its parent gets recapped.
![plvi 5](https://cloud.githubusercontent.com/assets/5634271/7130694/d85c0380-e242-11e4-8547-d8d91948857c.PNG)

I hope this helps.

**Comment by simality** [comment]
I just realized the the document and exhibit didn't actually recap.  So its probably a good thing that the notice didn't appear.  :-1:

**Comment by mlissner** [comment]
What OS are you running the extension on? The notifications can be pretty subtle in Chrome. You might have to look for them explicitly. Other than that, everything else here appears OK, right?

**Comment by simality** [comment]
Windows 7.  Professional at work.  Personal at home.

**Comment by mlissner** [comment]
I think they should show up in the notification drawer in the bottom right corner. Check there.

**Comment by simality** [comment]
Yeah I know...that's where they show in FF as well.  But they are,
anymore.  Are they showing up for you?

**Comment by mlissner** [comment]
I'm not on Windows, so not sure. They are working properly otherwise.

**Comment by simality** [comment]
I'll install the server this weekend....
